---
titre: "FAIR_Bioinfo: a turnkey training course and protocol for reproducible computational biology"
auteurs: "Thomas Denecker, Claire Toffano-Nioche"
DOI: "https://hal.archives-ouvertes.fr/hal-02880655"
journal: "HAL"
PMID: "" 
PMCID: "" 
annee: 2020
PDF: "FAIR_Bioinfo_Denecker_Toffano-Nioche.pdf"
---

### Abstract

Reproducibility plays an essential part in the success of a bioinformatics project. Indeed, Reproducibility makes it possible to guarantee the validity of scientific results and to simplify the dissemination of projects. To help disseminate Reproducibility principles among bioinformatics students, engineers and scientists, we created the FAIR_Bioinfo course, which presents a set of features we consider necessary to make a complete bioinformatics analysis reproducible. To illustrate the theoretical concepts of reproducibility, we use as an example a classic bioinformatics analysis (differential gene expression analysis from RNA-seq data). In short, we retrieve the data from public databases (ENA/SRA), we perform a reproducible analysis using a workflow management system (snakemake) in a virtual environment (Docker). The entire versioned (git) code is open source (Github https://github.com/thomasdenecker/FAIR_Bioinfo and dockerhub https://hub.docker.com/r/tdenecker/fair_bioinfo). The course book is available in English on GitBook (https://fair-bioinfo.gitbook.io/fair-bioinfo/) and the slides in French on Github. The visualization of the results is dynamic (Shiny app) and the PDF or HTML report (Rmarkdown) provides the results of the analysis and lists all user-selected parameters.
